client/scss/utilities/_mixins.scss

```diff
@@ -94,7 +128,14 @@
 @mixin css-grid-cells($columns, $key) {
   @for $i from 1 through $columns {
     .css-grid__cell--#{$key}#{$i} {
-      grid-column: span #{$i};
+      max-width: ($i / $columns) * 100%;
+      margin-bottom: 3 * $v-spacing-unit;
```

Comment:
Are we 100% defining this on every cell that is used throughout the site?
Just want to make sure we don't undo the good work done on spacingClasses.